src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const price = rental * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days < 7) {
+    return price - 20;
```

Comment:
Prefer using `const` over `let` where possible to avoid unintentional changes, as per checklist item 3. Although there are no `let` declarations here, it's a good practice to keep in mind for the future.